KProcessHacker/KProcessHacker.vcxproj

```diff
@@ -151,6 +203,19 @@
       <RandomizedBaseAddress>true</RandomizedBaseAddress>
     </Link>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">
+    <ClCompile>
+      <AdditionalIncludeDirectories>../phnt/include;../phlib/include;include;$(IntDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>POOL_NX_OPTIN;_WIN64;_ARM64_;ARM64;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
should this include _WIN64?

Reply:
According to [MSDN](https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019):
`_WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined.`